src/CommonGLPI.php

```diff
@@ -696,18 +700,59 @@ public static function displayStandardTab(CommonGLPI $item, $tab, $withtemplate
         return false;
     }
 
+    /**
+     * @param class-string<CommonGLPI>|null $form_itemtype
+     * @return string
+     */
+    private static function getTabIconClass(?string $form_itemtype = null): string
+    {
+        $default_icon = CommonDBTM::getIcon();
+        $icon = $default_icon;
+        $tab_itemtype = static::class;
+        $itemtype = $tab_itemtype;
+
+        if ($icon === $default_icon && is_subclass_of($tab_itemtype, CommonDBRelation::class)) {
```

Comment:
`$icon === $default_icon` will always be true here and in all checks of current method.